x-men/README.md

```diff
@@ -1,5 +1,9 @@
 # Superhero implementation
 
+# NEW INFO ON RUNNING THE EXTENSIONS
```

Comment:
I'd do a title here that indicates this is to integrate with the ai4sd extension